tools/integration/src/index.ts

```diff
@@ -1006,6 +1056,131 @@ function resolveDashboardReferences(dashboards: any[], dashboardsFolderPath: str
 	});
 }
 
+// Helper function to find entity by label
+async function findEntityByLabel(server: string, token: string, label: string, axiosInstance: any): Promise<any | null> {
+	try {
+		const entities = await getEntityList(server, token, axiosInstance);
```

Comment:
Can we ask backend REST API to return both the conflict label and id, then GET entity by id? This will be more efficient. We can take that as enhancement.